src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -1068,7 +1068,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         self ! CallFailedNonRetryable(collector, Seq.empty, None, e.getMessage)
       case Success(outputs) =>
         logger.info(s"Collection complete for Scattered Call ${collector.tag}.")
-        self ! CallCompleted(collector, outputs, Seq.empty, 0, hash = None, resultsClonedFrom = None)
+        self ! CallCompleted(collector, outputs, Seq.empty, ScriptReturnCode(0), hash = None, resultsClonedFrom = None)
```

Comment:
Same comment as above - is this wrongly assuming that a call completion implies return code 0?